packages/queries/src/query.test.ts

```diff
@@ -92,24 +92,6 @@ describe('queries', () => {
         };
     });
 
-    test('Find one actor by ID', () => {
```

Comment:
why did you remove tjos?

Reply:
Two identical tests... compare it to the one just above it

Reply:
Cool